From 954c9659cb2329045d6550723db51da612dd6ae8 Mon Sep 17 00:00:00 2001
From: ralmnsk <ralmnsk@gmail.com>
Date: Mon, 25 May 2020 00:00:49 +0300
Subject: [PATCH] "1File update git_1"

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 4083766..82642a3 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-line1
-line2
+line1 update
+line2 update
 line3
 line4
 line5
-- 
2.21.0.windows.1

